packages/react-core/src/components/Card/__tests__/CardHeader.test.tsx

```diff
@@ -1,5 +1,7 @@
 import { render, screen } from '@testing-library/react';
+import styles from '@patternfly/react-styles/css/components/Card/card';
 import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
```

Comment:
Thought I added this as a comment, but did you run into issues without this import? We shouldn't need it, so if we can let's remove it. Would you also mind removing the same import from the Toolbar test file? Didn't catch that in your PR until now 🙈

Reply:
I added it because VS Code gave a warning that `'toHaveClass' does not exist on type 'Matchers<void, HTMLElement>'` but the tests pass without it.

I'll remove it here and for toolbar.

Reply:
Let's keep it then, looks like we do have some other files that are importing that. We should probably go through at some point and make sure we're importing it in all these test files, but another problem for another day for another dev